From 2cdc5655dba77b3abdbd9989fb7b58e7058075c7 Mon Sep 17 00:00:00 2001
From: wata2ki <wata2ki@gmail.com>
Date: Wed, 12 Jun 2024 08:02:08 +0900
Subject: [PATCH 2/2] ARM64: dts: rockchip: enable can1 interface at a
 nanopc-t6 board

A nanopc-t6 board has 40-pin GPIO.  The can1 interface is
connected to pin 29-CAN_RX_M0 and 31-CAN_TX_M0.
This patch enable can1 interface at nanopc-t6 board.
---
 arch/arm64/boot/dts/rockchip/rk3588-nanopi6-rev01.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-nanopi6-rev01.dts b/arch/arm64/boot/dts/rockchip/rk3588-nanopi6-rev01.dts
index 14b683133321..1820e6cc93c2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-nanopi6-rev01.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-nanopi6-rev01.dts
@@ -783,3 +783,7 @@ &uart8 {
 	pinctrl-0 = <&uart8m1_xfer>;
 	status = "disabled";
 };
+
+&can1 {
+	status = "okay";
+};
-- 
2.25.1

